### DISCUSSION TITLE
There are no commands defined in the "kimai" namespace

### DISCUSSION THREAD

**Discussion by muzammil22** [kimai/kimai#2987] [On-Premise: Installation & Updates]
I deployed the app on shared hosting by creating a folder inside public_html. composer install worked fine.
Its a standard shared hosting where I host my Wordpress websites

I installed Symfony in my directory. Now when running " bin/console kimai:install -n" it returns
"There are no commands defined in the "kimai" namespace.:

Please advise.
Thanks.

**Comment by kevinpapst** [comment]
Here is everything you need: https://www.kimai.org/documentation/installation.html
Follow the instructions closely and it will work.

**Reply by muzammil22** [reply]
Thanks for the quick response.

I followed the one for Ionos since my hosting company wasn't present in the document. 

The hosting fulfils all the requirements.

I cloned from git
Ran composer install
Now stuck at kimai installation. 

It's returning this namespace error.

**Comment by muzammil22** [comment]
Does this only work on the hosting companies that you have mentioned ?

**Reply by kevinpapst** [reply]
No, "this" works "everywhere" with proper SSH access.
But nobody can help you with the information you shared, you have to learn how to ask questions while providing enough information.

**Comment by muzammil22** [comment]
Apologies for the confusion. I mistyped. 

The problem was solved. 

Thanks.

**Reply by kevinpapst** [reply]
Glad you solved it!
